src/KlaviyoAPI.php

```diff
@@ -420,13 +420,13 @@ protected function createRequestJson( array $params)
      * @param array $profiles
      * @throws KlaviyoException
      */
-    protected function checkProfile( array $profiles )
+    protected function checkProfile( $profiles )
     {
         foreach ( $profiles as $profile ) {
             if ( ! $profile instanceof ProfileModel ) {
                 throw new KlaviyoException( sprintf( " %s is not an instance of %s, You must identify the person by their email, using a \$email key, or a unique identifier, using a \$id.",
```

Comment:
You're going to get a warning here because there is only one arg being passed into the string. Let's just remove one of the string placeholders and hardcode it
```suggestion
                throw new KlaviyoException( sprintf( " %s is not an instance of ProfileModel, You must identify the person by their email, using a \$email key, or a unique identifier, using a \$id.",
```